### DISCUSSION TITLE
Two Window

### DISCUSSION THREAD

**Discussion by Decodetalkers** [slint-ui/slint#3310] [Q&A]
I want to export two root window to make two dialogs
like 
```slint
export component AppWindow2 inherits Window {}
export component AppWindow inherits Window {}
```
but I can only access AppWindow

**Comment by Decodetalkers** [comment]
the solution what I can come out is that make them two crates

**Comment by ogoffart** [comment]
This is tracked in https://github.com/slint-ui/slint/issues/784
